build-system/tasks/release/index.js

```diff
@@ -360,9 +360,20 @@ async function prependConfig_(outputDir) {
       type: channelConfig.type,
       ...require(`../../global-configs/${channelConfig.configBase}-config.json`),
     };
+    // Mapping of entry file names to a dictionary of AMP_CONFIG additions.
+    const targetsToConfig = MINIFIED_TARGETS.flatMap((minifiedTarget) => {
+      const targets = [];
+      if (!argv.esm) {
```

Comment:
I assume this covers `undefined` and `false`?

Reply:
Correct